notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.        fc = ds.sic_mean.isel(time=0).drop_vars("time").rename(dict(leadtime="time"))
Line #18.        fc['time'] = [pd.to_datetime(forecast_date) \
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; + dt.timedelta(days=int(e)) for e in fc.time.values]
</pre><p>These lines might benefit from some inline code comments to explain what is happening for users not adept at xarray/pandas.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>